Python/Product/ExternalProfilerDriver/VTuneInvoker.cs

```diff
@@ -0,0 +1,349 @@
+using System;
+using System.IO;
+using System.Linq;
+using System.Collections.Generic;
+using System.Text;
+
+using Microsoft.DotNet.PlatformAbstractions;
+
+namespace Microsoft.PythonTools.Profiling.ExternalProfilerDriver
+{
+    public class VTuneInvoker
+    {
+        private static readonly string _vtuneCl = @"\bin32\amplxe-cl.exe";
+        private static readonly string _vtune17Envvar = "VTUNE_AMPLIFIER_2017_DIR";
+        private static readonly string _vtune18Envvar = "VTUNE_AMPLIFIER_2018_DIR";
+
+        public static string VTunePath()
```

Comment:
Looks like this can be private and only used when someone passes null/empty to the constructor?